x-pack/plugins/observability_solution/synthetics/public/apps/embeddables/status_overview/status_overview_embeddable_factory.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import React, { useEffect } from 'react';
+import { DefaultEmbeddableApi, ReactEmbeddableFactory } from '@kbn/embeddable-plugin/public';
+import {
+  initializeTitles,
+  useBatchedPublishingSubjects,
+  fetch$,
+  PublishesWritablePanelTitle,
+  PublishesPanelTitle,
+  HasEditCapabilities,
+  SerializedTitles,
+} from '@kbn/presentation-publishing';
+import { BehaviorSubject, Subject } from 'rxjs';
+import type { StartServicesAccessor } from '@kbn/core-lifecycle-browser';
+import { STATUS_OVERVIEW_EMBEDDABLE } from '../constants';
+import { ClientPluginsStart } from '../../../plugin';
+import { StatusOverviewComponent } from './status_overview_component';
+
+export const getOverviewPanelTitle = () =>
+  i18n.translate('xpack.synthetics.statusOverview.displayName', {
+    defaultMessage: 'Synthetics Status Overview',
+  });
+
+export interface OverviewEmbeddableState extends SerializedTitles {
+  tags: string[];
+}
+
+export type StatusOverviewApi = DefaultEmbeddableApi<OverviewEmbeddableState> &
+  PublishesWritablePanelTitle &
+  PublishesPanelTitle &
+  HasEditCapabilities;
+
+export const getStatusOverviewEmbeddableFactory = (
+  getStartServices: StartServicesAccessor<ClientPluginsStart>
+) => {
+  const factory: ReactEmbeddableFactory<
+    OverviewEmbeddableState,
+    OverviewEmbeddableState,
+    StatusOverviewApi
+  > = {
+    type: STATUS_OVERVIEW_EMBEDDABLE,
+    deserializeState: (state) => {
+      return state.rawState as OverviewEmbeddableState;
+    },
+    buildEmbeddable: async (state, buildApi, uuid, parentApi) => {
+      // const [coreStart, pluginStart] = await getStartServices();
+
+      const { titlesApi, titleComparators, serializeTitles } = initializeTitles(state);
+      const defaultTitle$ = new BehaviorSubject<string | undefined>(getOverviewPanelTitle());
+      const tags$ = new BehaviorSubject(state.tags);
+      const reload$ = new Subject<boolean>();
+
+      const api = buildApi(
+        {
+          ...titlesApi,
+          isEditingEnabled: () => false,
```

Comment:
@shahzad31 The edit action appears by default in your embeddable. 

https://github.com/user-attachments/assets/bb8ba14c-52da-430d-93a6-ebf8b25f7ccd

In order to remove it you should completely remove following three properties:

```
isEditingEnabled: () => false,
getTypeDisplayName: () => getOverviewPanelTitle(),
onEdit: async () => {},
```